bin/ci-test

```diff
@@ -11,7 +11,7 @@ function print_usage() {
 function handle_args() {
   export SUITE_NAME=$1
   case $SUITE_NAME in
-    bundler1 | bundler2)
```

Comment:
do we still need the "in" check or can we simplify some more?

Reply:
I think we can keep that since in future we may also add bundler v3 in case. I tried to keep the syntax of supporting multiple as it is just we have one version, v2 now.

Reply:
No, not really.

Reply:
The condition are removed and it is simplified.